src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -197,6 +197,21 @@ configurationRegistry.registerConfiguration({
 				DeleteWordLeftTerminalAction.ID,
 				DeleteWordRightTerminalAction.ID
 			].sort()
+		},
+		'terminal.integrated.envVars.osx': {
+			'description': nls.localize('terminal.integrated.envVars.osx', "Object with environment variables that will be passed to the terminal on OS X"),
```

Comment:
The description should be clear that this is adding these to the environment, not replacing the entire environment